source/types.hpp

```diff
@@ -0,0 +1,4 @@
+#include <cstdint>
+
+typedef std::uint32_t Time;
```

Comment:
According to [conversation](https://github.com/Cloud-Storage-Team/algnet/pull/117#discussion_r2027908916), I think such comment is appropriate
```suggestion
// nanoseconds
typedef std::uint32_t Time;
```